discretize/TensorMesh.py

```diff
@@ -159,6 +159,22 @@ def gridN(self):
         """Nodal grid."""
         return self._getTensorGrid('N')
 
+    @property
+    def cellWidths(self):
+        """Cell widths of all cells in order."""
+
+        if self.dim == 1:
+            return self.h
+        if self.dim == 2:
+            hx = np.kron(np.ones(self.nCy), self.hx)
+            hy = np.kron(self.hy, np.ones(self.nCx))
+            return np.c_[hx, hy]
+        elif self.dim == 3:
+            hx = np.kron(np.ones(self.nCy*self.nCz), self.hx)
+            hy = np.kron(np.ones(self.nCz), np.kron(self.hy, np.ones(self.nCx)))
+            hz = np.kron(self.hz, np.ones(self.nCx*self.nCy))
+            return np.c_[hx, hy, hz]
```

Comment:
are there cases where you want to access these individually? (eg. hx, hy, hz) or are you always using all of them when asking for it?